src/data/nat/choose/sum.lean

```diff
@@ -103,6 +103,31 @@ begin
   simpa [sum_range_choose_halfway n] using t
 end
 
+lemma four_pow_le_two_mul_add_one_mul_central_binom (n : ℕ) :
+  4 ^ n ≤ (2 * n + 1) * choose (2 * n) n :=
+begin
+  calc 4 ^ n = (1 + 1) ^ (2 * n) : begin
+      rw one_add_one_eq_two,
+      rw pow_mul,
```

Comment:
Personally, I prefer this more condensed style:
```suggestion
      rw [one_add_one_eq_two, pow_mul],
```

Reply:
Untested, but I think that line of the `calc` bloc should just be true `by norm_num [pow_mul]`

Reply:
Yep, `by norm_num [pow_mul]` seems to work.